Parse/Internal/LocalDataStore/SQLite/PFSQLiteStatement.h

```diff
@@ -19,8 +19,9 @@ typedef struct sqlite3_stmt sqlite3_stmt;
 @interface PFSQLiteStatement : NSObject
 
 @property (atomic, assign, readonly) sqlite3_stmt *sqliteStatement;
+@property (atomic, strong, readonly) dispatch_queue_t databaseQueue;
```

Comment:
Do we really need this to be `atomic`? it's readonly and is set on initialization... :)